src/main/java/com/stuypulse/robot/commands/auton/PartnerBallAuton.java

```diff
@@ -0,0 +1,75 @@
+/************************ PROJECT DORCAS ************************/
+/* Copyright (c) 2022 StuyPulse Robotics. All rights reserved.  */
+/* This work is licensed under the terms of the MIT license.    */
+/****************************************************************/
+
+package com.stuypulse.robot.commands.auton;
+
+import com.stuypulse.robot.RobotContainer;
+import com.stuypulse.robot.commands.conveyor.ConveyorShootCommand;
+import com.stuypulse.robot.commands.drivetrain.DrivetrainAlignCommand;
+import com.stuypulse.robot.commands.drivetrain.DrivetrainRamseteCommand;
+import com.stuypulse.robot.commands.intake.IntakeAcquireForeverCommand;
+import com.stuypulse.robot.commands.intake.IntakeExtendCommand;
+import com.stuypulse.robot.commands.leds.LEDSetCommand;
+import com.stuypulse.robot.commands.shooter.ShooterFenderShotCommand;
+import com.stuypulse.robot.commands.shooter.ShooterRingShotCommand;
+import com.stuypulse.robot.constants.Settings.Limelight;
+import com.stuypulse.robot.util.LEDColor;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+/*-
+ * @author Vincent Wang
+ * @author Shaurya Sen
+ */
+ 
+public class PartnerBallAuton extends SequentialCommandGroup {
+    // Time it takes for the shooter to reach the target speed
+    private static final double SHOOTER_INITIALIZE_DELAY = 0.3;
+    // Time it takes for shooter to go from Fender to Ring shot speed
+    private static final double SHOOTER_TO_RING_DELAY = 0.2;
+    // Time it takes for the conveyor to give the shooter the ball
+    private static final double CONVEYOR_TO_SHOOTER = 3.0;
+    // Time we want to give the drivetrain to align
+    private static final double DRIVETRAIN_ALIGN_TIME = 3.0;
+    // Time to acquire partner auton ball
+    private static final double ACQUIRE_PARTNER_BALL_TIME = 5.0;
```

Comment:
We might want to consider increasing this time during testing, or also maybe consider adding a start delay at the beginning to prevent us from running into alliance partners.